apps/deploy-web/src/components/onboarding/OnboardingContainer/OnboardingContainer.tsx

```diff
@@ -67,7 +67,8 @@ export const OnboardingContainer: React.FunctionComponent<OnboardingContainerPro
   const router = d.useRouter();
   const { user } = d.useUser();
   const { data: paymentMethods = [] } = d.usePaymentMethodsQuery({ enabled: !!user?.stripeCustomerId });
-  const { analyticsService, urlService, authService, chainApiHttpClient, deploymentLocalStorage, appConfig, errorHandler } = d.useServices();
+  const { analyticsService, urlService, authService, chainApiHttpClient, deploymentLocalStorage, appConfig, errorHandler, windowLocation, windowHistory } =
```

Comment:
non-blocking: perhaps it makes more sense to just use `window` so that we wouldn't need to add every window prop in the future? 

```suggestion
  const { analyticsService, urlService, authService, chainApiHttpClient, deploymentLocalStorage, appConfig, errorHandler, windowLocation, window } =
```

Reply:
from another hand, it will be harder to mock the whole `window` in tests. Though possible with `mockDeep<Window>()`. Honestly, I think this is a rare case, for example localStorage we need to encapsulate separately.

but the main reason behind this change is that I just needed to fix it for new jsdom version which doesn't allow to redefine `location` and `history` on `window`. Ideally we should use router for both cases, not sure whether react router exposes this information